code/modules/mob/living/simple_mob/subtypes/animal/giant_spider/frost.dm

```diff
@@ -35,3 +35,6 @@
 /mob/living/simple_mob/animal/giant_spider/frost/sif
 	desc = "Icy and blue, it makes you shudder to look at it. This one has brilliant blue eyes. \
 	It isn't native to Sif."
+
+/mob/living/simple_mob/animal/giant_spider/frost/event
```

Comment:
Add //YW addition to top of this line. Or put this new animal in a new file called frost_yw.dm